internal/summoner/acquire/headlessNG.go

```diff
@@ -238,7 +238,7 @@ func PageRender(v1 *viper.Viper, mc *minio.Client, timeout time.Duration, url, k
 	expression := `
 		function getMetadata() {
 			return new Promise((resolve, reject) => {
-				const elements = document.querySelectorAll('script[type="application/ld+json"]');
+				const elements = document.querySelectorAll('script[type=JSONContentType]');
```

Comment:
Is this going to work. Where does this value get passed into headless chrome?

Reply:
WHEW, thank you, well-spotted. That is NOT going to work as-is.